Templates/GitlabCIPipeline-for-distributed-runner/.gitlab-ci.yml

```diff
@@ -289,14 +256,60 @@ Packaging:
         - if: $CI_COMMIT_BRANCH =~ /release\/.*/  && $pipelineType != "preview"
         - if: $CI_COMMIT_BRANCH =~ /epic\/.*/  && $pipelineType != "preview"
     script: |
+        echo [INFO] Start deployment plan generation job
+
+        if [ ${pipelineType} == "release" ]; then
+            echo [INFO] Start computing the next release version.
+            echo [INFO]          Pipeline Type: ${pipelineType}
+            echo [INFO]           Release Type: ${releaseType}
+                
+            # Call script to compute the next release version from baselineReference file.
+            stringOut=$(zowe rse issue unix-shell ". ~/.bash_profile && computeReleaseVersion.sh -w ${uniqueWorkspaceId} -a ${application} -b ${CI_COMMIT_REF_NAME} -r ${releaseType}" --user ${RSEAPI_USER} --password ${RSEAPI_PASSWORD} --cwd ${RSEAPI_WORKING_DIRECTORY})
+            RC=$?
 
-        zowe rse issue unix-shell ". ~/.bash_profile && wazideploy-generate.sh -w ${uniqueWorkspaceId} -i ${application}.tar" --user ${RSEAPI_USER} --password ${RSEAPI_PASSWORD} --cwd ${RSEAPI_WORKING_DIRECTORY}
-        RC=$?
-        if [ ${RC} -eq 0 ]; then
-            echo [INFO] Generate deployment plan job passed.
+            if [ ${RC} -ne 0 ]; then
+                echo [ERROR] Failed to compute the next release version. Exit code: ${RC}.
+                echo ${stringOut}
+            fi
+                
+            # Extract the next release version.
+            export releaseVersion=$(echo "${stringOut}" | grep "The next release version:" | grep -oE '[0-9]+\.[0-9]+\.[0-9]+')
+                    
+            if [ "${releaseVersion}" == "" ]; then
+                RC=8
+                echo [ERROR] Failed to extract the release version from the output of computeReleaseVersion script. Exit code: ${RC}.
+                echo ${stringOut}
+            fi
+
```

Comment:
The below block - Line 284-291 should be surrounded by

```
if [ ${RC} -eq 0 ]; then
fi
``` 

to only execute it, once the previous steps were successful.